components/style/properties/longhand/svg.mako.rs

```diff
@@ -15,12 +15,27 @@ ${helpers.single_keyword("vector-effect", "none non-scaling-stroke", products="g
 
 // Section 13 - Gradients and Patterns
 
+${helpers.predefined_type(
+    "stop-color", "CSSColor",
+    "::cssparser::Color::RGBA(::cssparser::RGBA { red: 0., green: 0., blue: 0., alpha: 1. })",
```

Comment:
I think we should have a |use| somewhere for RGBA so that we don't need to have all this junk in the line initializer.

Also, the |0.| stuff looks weird, and I don't see it elsewhere. Any reason not to just do 0 or 0.0? Here and below.